src/components/user/UserMenu.js

```diff
@@ -3,9 +3,11 @@ import { Sidebar, Image, Segment, Menu, Icon } from "semantic-ui-react";
 import { UserContext } from "contexts/UserContext";
 import { avatarForName } from "games/kalambury/utils/avatar";
 import { useTranslation } from "react-i18next";
+import { useQuery } from "react-query";
 
 const UserMenu = ({ children }) => {
-  const { user, logout } = useContext(UserContext);
+  const { user, logout, getNickName } = useContext(UserContext);
+  const { data } = useQuery(["user-nickname", user.uid], (key, id) => getNickName(id));
```

Comment:
Even after I log-in using email I'm shown as `Guest`

Reply:
I think it's because you have old account, I'm cleaning all accounts in firebase

Reply:
I deleted all of the accounts, with this branch and new account it should work :D